test/e2e_node/topology_manager_test.go

```diff
@@ -125,7 +125,7 @@ func getMinRemoteDistanceForNode(nodeToDistances map[int][]int) int {
 func detectNUMADistances(numaNodes int) map[int][]int {
```

Comment:
could you please add `ginkgo.GinkgoHelper()` as first statement in the function? this is meant to get nicer and more helpful stacktraces on error

Reply:
added